python/paddle/v2/fluid/tests/test_ctc_greedy_decode.py

```diff
@@ -0,0 +1,56 @@
+import sys
+import unittest
+import numpy as np
+from op_test import OpTest
+from test_softmax_op import stable_softmax
+
+
+def CTCGreedyDecode(softmax, blank, merge_repeated):
+    prev_token = -1
+    result = []
+    for token in np.argmax(softmax, axis=1):
+        if (token != blank) and not (merge_repeated and token == prev_token):
+            result.append(token)
```

Comment:
Should there be one line `prev_token = token`?

Reply:
Thx. Fixed.